zeppelin-server/src/main/java/org/apache/zeppelin/server/ZeppelinServer.java

```diff
@@ -150,27 +147,50 @@ public static void main(String[] args) throws InterruptedException {
   }
 
   private static Server setupJettyServer(ZeppelinConfiguration conf) {
-    AbstractConnector connector;
+
+    final Server server = new Server();
+    ServerConnector connector;
+
     if (conf.useSsl()) {
-      connector = new SslSelectChannelConnector(getSslContextFactory(conf));
+
+      HttpConfiguration http_config = new HttpConfiguration();
```

Comment:
Could you please change `http_config` to httpConfig? And also do `https_config` to `httpsConfig`?